assets/js/samples.js

```diff
@@ -0,0 +1,9 @@
+export default function() {
+  document.querySelectorAll("button.play").forEach(function (button) {
+    button.addEventListener("click", function(e){
+      const sampleName = button.dataset.sample;
+      const media = document.querySelector(`audio#${sampleName}`);
+      media.play();
+    });
+  });
```

Comment:
If you play a sample while another one is playing, then they play on top of each other and you can't hear the sample clearly! I've added this code which should stop the current sample when another one is played :)

```suggestion
  var currently_playing = null;
  document.querySelectorAll("button.play").forEach(function (button) {
    button.addEventListener("click", function(e){
      if (currently_playing != null) {
        currently_playing.pause();
        currently_playing.currentTime = 0;
      }
      const sampleName = button.dataset.sample;
      const media = document.querySelector(`audio#${sampleName}`);
      currently_playing = media;
      media.play();
    });
  });
```

Reply:
Fair enough, though I was only going by the existing behaviour on the samples pages in the app - they are allowed to overlap there.
In some ways, I feel that overlapping samples are not bad - if people want to hear samples clearly, they can be patient and wait till they finish 😂  though I suppose this may be worth thinking about if we end up providing samples that are more than about 10 seconds long... @samaaron your thoughts on this?